### ISSUE TITLE
Github Actions failing on ubuntu-latest

### ISSUE THREAD

**Issue by JacquesCarette** [JacquesCarette/Drasil#3181]
See [this build failure](https://github.com/JacquesCarette/Drasil/actions/runs/3608923700/jobs/6081806688) for an example. Seems to be related to lualatex and fonts? Might be new, even though this bug seemed to appear in the past (on the lualatex side, not ours).

Sam T.-H. on twitter heartily recommends using [ACT](https://github.com/nektos/act) for locally debugging such things.

**Comment by smiths** [comment]
ACT does look like a good option to run GitHub Actions locally.  The README for ACT doesn't mention another advantage that occurs to me, running the Actions locally doesn't use up the free minutes.  When we last spoke about the free minutes, there wasn't much worry that we were going to exceed them, but it is still nice to have a local option, in case there is a concern in the future.

**Comment by balacij** [comment]
I'm very surprised these errors are popping up now, only after GH updated their image to 22.04. I'm using NixOS with unstable packages (most packages are newer than Ubuntu's), and I have never seen any noticeable errors like these.

I don't think the lualatex fonts thing is an "error," but a warning message. I have it appear on my machine as well. However, the bit about "$TERM" is odd. I think you fixed that now though.

ACT looks like a great tool for debugging the CI!

**Comment by balacij** [comment]
Relevant: https://github.com/JacquesCarette/Drasil/pull/5271#discussion_r3545456430

There's not much we can do to debug the errors back in 2022 and it looks like we're at the mercy of GitHub and its provided package sets here. We haven't had issue with tex other than build times, but that's being worked on in #5271 . I think we can close this issue because there's not much we can really do here other than wait for the next issue to pop up and try to deal with it.
